### PR TITLE
Make tests run (including on TravisCI and CircleCI)

### PR DESCRIPTION
**Pull request by noirbizarre**
This pull request should fix the karma tests and make them runnning on TravisCI et CircleCI.

- [x] Make tests run localy
- [x] Make test run on TravisCI
- [x] Make test run on CircleCI
- [x] Make all tests pass
- [x] Update documentation

### CODE REVIEW

File: specs/plugins/router.specs.js [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
-    //     template: '{{{text | markdown}}}'
-    // });
-
-    // function tester(text) {
-    //     return new Tester({
-    //         el: '#sandbox',
-    //         data: {
-    //             text: text
-    //         }
+    // var Vue = require('vue');
```
Comment (davidbgk) [inline specs/plugins/router.specs.js]: Remove?

File: tasks.py [hunk 2]
```diff
@@ -54,26 +54,18 @@ def cover():
...
+    cmd = 'npm run -s test:{0}'.format('watch' if watch else 'unit')
+    lrun(cmd, pty=True)
 
 
 @task
 def doc():
     '''Build the documentation'''
     header('Building documentation')
-    lrun('cd doc && make html', pty=True)
+    lrun('mkdocs serve', pty=True)
```
Comment (davidbgk) [inline tasks.py]: Maybe update the doc to use inv instead of mkdocs in that case?

File: specs/plugins/router.specs.js (hunk 1 above)
Comment (noirbizarre) [inline specs/plugins/router.specs.js]: Yes, I forgot to remove empty test files

File: tasks.py (hunk 2 above)
Comment (noirbizarre) [inline tasks.py]: :+1:

File: js/components/form/base-form.js [hunk 3]
```diff
@@ -75,11 +74,11 @@ export default {
                     return;
                 }
 
-                let properties = field.id.split('.'),
-                    currentSchema = schema,
-                    required = true,
-                    path = '',
-                    prop;
+                const properties = field.id.split('.');
+                let currentSchema = schema;
```
Comment (davidbgk) [inline js/components/form/base-form.js]: So it's a trend :smile:

File: js/components/form/markdown-editor.vue [hunk 4]
```diff
@@ -95,7 +96,7 @@ export default {
                     name: 'extras',
                     data: [{
                         name: 'btnSummary',
-                        title: Vue._('Summary'),
+                        title: this._('Summary'),
```
Comment (davidbgk) [inline js/components/form/markdown-editor.vue]: `this` required here?

File: js/config.js [hunk 5]
```diff
@@ -25,6 +25,15 @@ function _link(rel) {
     return _attr(`link[rel=${rel}]`, 'href')
 }
 
+/**
+ * A simple helper to parse JSON from a <meta/> tag
+ * @return {[type]} [description]
+ */
+function _jsonMeta(name) {
+    const data =  _meta(name);
```
Comment (davidbgk) [inline js/config.js]: Two spaces.

File: karma.conf.js [hunk 6]
```diff
@@ -28,74 +15,98 @@ module.exports = function(config) {
...
 
-
-        // enable / disable watching file and executing tests whenever any file changes
-        // autoWatch: true,
-
-
         // start these browsers
         // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-        // browsers: ['Chrome'], //, 'Firefox', 'PhantomJS'],
+        // browsers: ['Chrome', 'Firefox', 'PhantomJS'],
```
Comment (davidbgk) [inline karma.conf.js]: Weird change ;)

File: js/components/form/base-form.js (hunk 3 above)
Comment (noirbizarre) [inline js/components/form/base-form.js]: Yes, I remembered so I applied this rule every where ;)

File: js/components/form/markdown-editor.vue (hunk 4 above)
Comment (noirbizarre) [inline js/components/form/markdown-editor.vue]: It prevent importing `Vue` as `Vue` instances already have the same helper.

**Comment by davidbgk** [comment]
Terrific work, r+wc!

File: js/components/form/markdown-editor.vue (hunk 4 above)
Comment (davidbgk) [inline js/components/form/markdown-editor.vue]: You can't just use the imported `_` directly?

File: karma.conf.js (hunk 6 above)
Comment (noirbizarre) [inline karma.conf.js]: Just to properly list the possible options ;)

File: js/components/form/markdown-editor.vue (hunk 4 above)
Comment (noirbizarre) [inline js/components/form/markdown-editor.vue]: Oops, I didn't see it was already imported !

**Pull request merged**